app/routes/entries/$headword/edit/QuotationAddingForm.tsx

```diff
@@ -1,26 +1,33 @@
 import { resetFetcher } from "~/routes/api/reset-fetcher"
-import { Form, useFetcher } from "@remix-run/react"
+import { Form, Link, useFetcher } from "@remix-run/react"
+import { DefaultErrorBoundary } from "~/components/elements/DefaultErrorBoundary"
 import type { CitationSearchLoaderData } from "~/routes/api/citations/$searchTerm[.json]"
 import BankInput from "~/components/bank/BankInput"
 import LabelledField from "~/components/bank/LabelledField"
 import BankRadioOrCheckbox from "~/components/bank/BankRadioOrCheckbox"
 import Button from "~/components/elements/Button"
 import { attributeEnum } from "~/components/editing/attributeEnum"
+import { useState } from "react"
 
 interface QuotationAddingFormProps {
   meaningId: number
 }
 
 const citationSearchUrl = (
   searchTerm: string,
-  { orderBy, orderDirection }: { orderBy?: string; orderDirection?: string }
+  {
+    orderBy,
+    orderDirection,
+    page,
+  }: { orderBy?: string; orderDirection?: string; page?: string }
 ) => {
   const url = `/api/citations/${searchTerm}.json`
 
   const searchParams = new URLSearchParams()
-  searchParams.set("searchField", "headword")
+  searchParams.set("searchField", "headword") // should this be headword or citation?
```

Comment:
Headword, I think, because we want to find citations that have been associated with the headword being edited in the DET.